src/raftstore/store/worker/pd.rs

```diff
@@ -200,7 +307,17 @@ impl<T: PdClient> Runner<T> {
         router: RaftRouter,
         db: Arc<DB>,
         scheduler: Scheduler<Task>,
+        store_heartbeat_interval: u64,
     ) -> Runner<T> {
+        let mut monitor_interval = store_heartbeat_interval / 2;
+        if monitor_interval == 0 {
+            monitor_interval = 1;
```

Comment:
It means both `store_heartbeat_interval` and `monitor_interval` are `1s`? Why not just `500ms`?